lib/src/bot.dart

```diff
@@ -325,4 +326,22 @@ class Bot {
 
     await telegram.sendMessage(chatId, joke.joke);
   }
-}
+  void _sendRealMusic(TeleDartMessage message) async {
+    if (message.text == null) {
+      await message.reply('Нахуй пошол, мудило!!1');
+      return;
+    }
+
+    var rawText = message.text.split(' ');
```

Comment:
I also suggest adding a check that the link that the user has sent is indeed leads to YouTube music. Something like .startsWith('music.youtube.com')